File: python/2455.py
p1, c1, p2, c2 = map(int, input().split(' '))
calc = p1 * c1 == p2 * c2;

if (p1 * c1 == p2 * c2):
    print(0)
elif p1 * c1 > p2 * c2:
    print(-1)
else:
    print(1)

File: javascript/2455.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var [p1, c1, p2, c2] = input.split(' ').map(e => parseInt(e))

var calc = p1 * c1 == p2 * c2
    
    if(p1 * c1 == p2 * c2) console.log(0)
    else if(p1 * c1 > p2 * c2) console.log(-1)
    else console.log(1);